gcip/addons/kaniko/jobs.py

```diff
@@ -12,7 +12,7 @@
 
 
 def execute(
-    gitlab_executor_image: Optional[Union[Image, str]] = Image("gcr.io/kaniko-project/executor:debug", entrypoint=[""]),
+    gitlab_executor_image: Optional[Union[Image, str]] = Image("gcr.io/kaniko-project/executor:debug", entrypoint=["sh"]),
```

Comment:
I think the entrypoint should be empty for gitlab-ci.
https://docs.gitlab.com/ee/ci/docker/using_kaniko.html#building-a-docker-image-with-kaniko